src/context-providers/builder/hook.ts

```diff
@@ -120,7 +120,7 @@ export const useBuilder = () => {
         [state.orderedIdentifiers, state.elements, state.mode]
     );
 
-    const focusElement = useCallback((element: TElement, isDirty?: boolean) => {
+    const focusElement = useCallback((element: TElement, isDirty = false) => {
```

Comment:
remove it as I dont see any need to set isDirty anymore
```suggestion
    const focusElement = useCallback((element: TElement) => {
```